erts/emulator/drivers/common/inet_drv.c

```diff
@@ -2610,7 +2612,7 @@ http_header_inetdrv(void* arg, const http_atom_t* name, const char* name_ptr,
       i = LOAD_INT(spec, i,  0);
       i = http_load_string(desc, spec, i, name_ptr, name_len);
     }
-    i = LOAD_ATOM(spec, i, am_undefined);
+    i = http_load_string(desc, spec, i, oname_ptr, oname_len);
     i = http_load_string(desc, spec, i, value_ptr, value_len);
     i = LOAD_TUPLE(spec, i, 5);
 
```

Comment:
The `ASSERT(i <= 26)` a couple of lines down needs to be updated.

Reply:
Also the `spec` buffer needs to be adjusted to fit the new data.

Reply:
@garazdawi - In my understanding the new `26` should be `27`. Previously the `undefined` atom needed 2 placeholders, now the `list` or `binary` data need 3 places in the `spec` buffer. Hope this logic is correct.

Reply:
yes, that seems to be correct.